### PR TITLE
Create Egg2

### PR DESCRIPTION
**Pull request by niyu2003**
Making a new pull request since the previous one got closed.

### CODE REVIEW

File: Egg2 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        System.out.println("");
+        Top();
+        Bottom();
+        Middle();
+        Bottom();
+        System.out.println("");
+        Top();
+        Middle();
+        Bottom();
+
```
Comment (niyu2003) [inline Egg2]: there should be a } (close bracket) here.

File: Egg2 [hunk 2]
```diff
@@ -0,0 +1,29 @@
+public class Egg2 {
+    public static void main(String[] args) {
+        Top();
+        Bottom();
+        Middle();
+        System.out.println("");
```
Comment (niyu2003) [inline Egg2]: this line should be removed

File: Egg2 [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        Top();
+        Bottom();
+        Middle();
+        Bottom();
+        System.out.println("");
```
Comment (niyu2003) [inline Egg2]: this line should be removed.

**Comment by twang** [comment]
Thanks @niyu2003!

I'm guessing you couldn't see this "Reopen" button on the other pull request https://github.com/twangcs/unit-1-work-in-progress-Lilyjiang6/pull/1 so you opened this one?

image

**Comment by niyu2003** [comment]
Huh, interesting. That "reopen pull request button actually isn't popping up for me: 
image

This was why I created this new pull request.

**Pull request closed without merging**